### DISCUSSION TITLE
No -all.jar with version 1.9.0

### DISCUSSION THREAD

**Discussion by rbraeunlich** [open-telemetry/opentelemetry-java-instrumentation#4720] [Q&A] [answered]
Hello everyone,
I saw that yesterday version 1.9.0 has been released.
The release no longer contains a -all.jar, as the previous ones did. I couldn't find any hint in the release notes if this was intentional.
Some of my Docker image pipelines crash because I used the `/lastest/download/*-all.jar` URL.
Could you please tell me if it is missing or no longer published? And if the latter is the case, does the "normal" JAR now provide the same functionality?
Cheers,
Ronny

**Comment by rbraeunlich** [comment, marked as answer]
Dang, found it in the 1.7.0 release notes. Sorry folks
https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.7.0

**Reply by trask** [reply]
no worries, glad u found it 👍
